README.rst

```diff
@@ -87,3 +87,19 @@ Follow the same instructions as above except for the following:
 3. **Copy the OpenSSL folder**
 
 4. **Build the amalgamation and install with the latest Python x64**
+
+SQLCipher compatibility issues
+----------------------------
+The encryption has default compatibility with SQLCipher version, installed on your machine (current SQLCipher version 4).
+You have to pass the ``PRAGMA cipher_compatibility = 3`` before doing any operations on a database, crypted with SQLCipher version 3, when version 4 is installed.
```

Comment:
"You have to execute ``PRAGMA cipher_compatibility = 3`` before doing any operations on a database encrypted with SQLCipher version 3 when a newer version is installed."